Add function to fade out not-highlight elements.

seigodev opened the pull request:
issue: Add a graph user interface. #33
強調表示されていない要素(ノード、エッジ)を薄く表示する関数を追加しました。

seigodev committed:
Add function to fade out not-highlight elements.

強調表示されていない要素(ノード、エッジ)を薄く表示する関数を追加しました。

aabaa commented on demo/draw_graph.js:
```diff
@@ -276,3 +276,14 @@ function highlight_connected_elements(cy, generations, select_node, is_ancestor)
+function fade_out_not_highlight_elements(cy){
+    let other = cy.elements();
+    other = other.difference(cy.elements(".highlight"));
+    cy.$(other).addClass("not_highlight");
```
クラス名はnot_highlightではなく，「目立たなく(薄く表示)する」ことをダイレクトに表現する名前に変更してください．

seigodev committed:
Rename not-highlight class.

クラス名not_highlightをクラス名fadedに変更しました。
not_highlight  ->  faded
関数名も少し変更しました。
fade_out_not_highlight_elements -> fade_not_highlight_elements
